Orange/widgets/visualize/owscatterplot.py

```diff
@@ -425,7 +425,8 @@ def update_graph(self, reset_view=True, **_):
         self.graph.update_data(self.attr_x, self.attr_y, reset_view)
 
     def selection_changed(self):
-        self.send_data()
+        if self.auto_send_selection:
+            self.send_data()
```

Comment:
This half-reimplements the logic of `Orange.widgets.gui.auto_commit`. This does not work:

- disable auto commit,
- select some points,
- enable auto commit.

Selected points are not sent.

The `auto_commit` function has a local variable `dirty`, which you would have to sent, but can't access from here.

A solution that would work would be to simply call `self.commit` instead of directly calling `self.send_data`. A (slight) problem with this is that it would re-send features when the selection is changed. This could be prevented by storing the last sent features and not sending them when not needed. Not most elegant solution, though.

Reply:
Fixed.